test/schedule_manager_test.exs

```diff
@@ -28,72 +29,67 @@ defmodule Verk.ScheduleManagerTest do
   test "handle_info :fetch_retryable without jobs to retry", %{ script: script } do
     state = %State{ redis: :redis }
     now = :now
-    expect(Timex.Time, :now, [:seconds], now)
     expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now]], {:ok, nil})
 
     assert handle_info(:fetch_retryable, state) == { :noreply, state }
     assert_receive :fetch_retryable
 
-    assert validate [Timex.Time, Redix]
+    assert validate [ Redix]
   end
 
-  test "handle_info :fetch_retryable with jobs to retry", %{ script: script } do
+  test "handle_info :fetch_retryable with jobs to retry", %{ script: script  } do
     state = %State{ redis: :redis }
-    now = :now
+    now = Time.now
```

Comment:
`Verk.Time.now` just delegates to that implementation. In general I lean towards using the interface in the library itself (e.g. `Verk.Time`) rather than an outside implementation (even one that's nominally part of the standard library sometimes) so that there's a layer of safety around API changes and it gives the tests an explicit, internally consistent contract to work with.